src/custom/utils/index.ts

```diff
@@ -1,4 +1,7 @@
 import { ChainId } from '@uniswap/sdk'
+import { getExplorerOrderLink } from './explorer'
+
+const GP_ORDER_ID_LENGTH = 114 // 112 (56 bytes in hex) + 2 (it's prefixed with "0x")
```

Comment:
That looks hackish as hell =P

Reply:
hahha, indeed :) 
happy if you want to supersede

Reply:
Not at all, all yours